en/docs/reference/mediators/dss-mediator.md

```diff
@@ -4,7 +4,6 @@ The **Data Service Call Mediator** is used to invoke data service operations. It
 
 !!! Info
     - You need to first have a [Data Service Project]({{base_path}}/develop/creating-artifacts/data-services/creating-data-services) to use the Data Service Call mediator.
```

Comment:
Since there's only one point inside this note, shall we remove `-` from here?
```suggestion
    You need to first have a [Data Service Project]({{base_path}}/develop/creating-artifacts/data-services/creating-data-services) to use the Data Service Call mediator.
```